src/Eventuras.Domain/EventInfoOptions.cs

```diff
@@ -0,0 +1,14 @@
+﻿#nullable enable
+
+namespace Eventuras.Domain;
+
+public class EventInfoOptions
+{
+    public EventInfoRegistrationPolicy RegistrationPolicy { get; set; } = new();
+
+    public class EventInfoRegistrationPolicy
+    {
+        public int? AllowedRegistrationEditHours { get; set; } = null;
```

Comment:
Could we default this to 24?

Reply:
Breaking change, same as with `AllowModificationsBeforeLastCancellationDate`

Reply:
Fixed